espnet2/spk/layers/RawNetBasicBlock.py

```diff
@@ -0,0 +1,136 @@
+import math

```

Comment:
Can you rename the file with lower cases? It's better to keep consistency with other files.